### PR TITLE
Con 2142 summary update

### PR DESCRIPTION
**Pull request by VasanthaKasirajan3008**

### CODE REVIEW

File: src/SFA.DAS.AssessorService.Database/StoredProcedures/OppFinder_Update_StandardSummary.sql [hunk 1]
```diff
@@ -146,11 +146,15 @@ BEGIN
 	AND ex1.StandardName IS NULL
 	GROUP BY StandardCode, Total.StandardReference, Total.StandardName, Sector, StandardLevel, Region,Ordering 
 	ORDER BY StandardCode, Ordering;
+
+	RAISERROR ('Error raised in store procedure OppFinder_Update_StandardSummary TRY block.', 15, 1);
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.AssessorService.Database/StoredProcedures/OppFinder_Update_StandardSummary.sql:150]: Does this not just raise an error even if the procedure succeeds?

File: src/SFA.DAS.AssessorService.Database/StoredProcedures/OppFinder_Update_StandardSummary.sql [hunk 2]
```diff
@@ -146,11 +146,15 @@ BEGIN
...
 	GROUP BY StandardCode, Total.StandardReference, Total.StandardName, Sector, StandardLevel, Region,Ordering 
 	ORDER BY StandardCode, Ordering;
+
+	RAISERROR ('Error raised in store procedure OppFinder_Update_StandardSummary TRY block.', 15, 1);  
 	
 	END TRY
 	BEGIN CATCH;
 		-- Some basic error handling
 		SELECT @Error_Code = ERROR_NUMBER(), @Error_Message = ERROR_MESSAGE();
+		RAISERROR(@Error_Message, 15, 1) WITH NOWAIT;
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.AssessorService.Database/StoredProcedures/OppFinder_Update_StandardSummary.sql:156]: Does this actually work the error message being shown is constructed on the line following this statement?

File: src/SFA.DAS.AssessorService.Database/StoredProcedures/OppFinder_Update_StandardSummary.sql [hunk 3]
```diff
@@ -146,11 +146,15 @@ BEGIN
 	AND ex1.StandardName IS NULL
 	GROUP BY StandardCode, Total.StandardReference, Total.StandardName, Sector, StandardLevel, Region,Ordering 
 	ORDER BY StandardCode, Ordering;
+
+	RAISERROR ('Error raised in store procedure OppFinder_Update_StandardSummary TRY block.', 15, 1);  
 	
 	END TRY
 	BEGIN CATCH;
 		-- Some basic error handling
 		SELECT @Error_Code = ERROR_NUMBER(), @Error_Message = ERROR_MESSAGE();
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.AssessorService.Database/StoredProcedures/OppFinder_Update_StandardSummary.sql:155]: Is this bit needed now?

File: src/SFA.DAS.AssessorService.Database/StoredProcedures/OppFinder_Update_StandardSummary.sql [hunk 4]
```diff
@@ -146,11 +146,15 @@ BEGIN
...
+		RAISERROR(@Error_Message, 15, 1) WITH NOWAIT;
+		SET @Error_Message = ERROR_NUMBER() + ERROR_LINE() + ERROR_MESSAGE()
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.AssessorService.Database/StoredProcedures/OppFinder_Update_StandardSummary.sql:157]: What does this look like in Kibana - is the formatting helpful - are there any spaces in the string between the number and the line?

File: src/SFA.DAS.AssessorService.Application.Api/Controllers/OppFinder/OppFinderController.cs [hunk 5]
```diff
@@ -63,7 +64,11 @@ public async Task UpdateStandardSummary([FromBody] UpdateStandardSummaryRequest
                 _logger.LogInformation($"Received request to update standard summary");
                 await _mediator.Send(request);
             }
-            catch(Exception ex)
+            catch (SqlException sqlEx)
+            {
+                _logger.LogError($"Request to update standard summary failed : {sqlEx.Message}");
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.AssessorService.Application.Api/Controllers/OppFinder/OppFinderController.cs:69]: I think the actually error can be log as well possibly giving more details

**Review by ChrisJWoodcock — CHANGES_REQUESTED** [review]

**Comment by VasanthaKasirajan3008** [comment]
Not able to replicate, so we dont need this PR

**Pull request closed without merging**